Containerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.12
+
+RUN apt-get update
+RUN apt-get install -y r-base
```

Comment:
This could be somewhat risky as you are picking whatever is the current version of r-base supported by apt-get. Why not point to a fixed version instead? Also, why not using one of the rocker images instead?